low-level/message_handlers/node_data_msg_handler.py

```diff
@@ -564,6 +573,23 @@ def _generate_cpu_data(self):
             self._write_internal_msgQ(RabbitMQegressProcessor.name(), jsonMsg)
             self.cpu_fault = False
 
+    def _check_ping_to(self, ip):

```

Comment:
can we keep this function in cortx-utils as its generic function

rename to check_ip_accessibility()

Reply:
I think hostname (srvnode-1) is being passed here. may be `check_hostname_accessibility`?